### DISCUSSION TITLE
User Journey Discussion

### DISCUSSION THREAD

**Discussion by manishapriya94** [ProgramEquity/amplify#57] [General]
## Goals
- Refresh content copy: echo the knowledge and values found across our education journey for users while they're taking action 

**Resources:** 
- [Research Brief ](https://www.notion.so/programequity/Dare-to-Dream-Civic-Engagement-is-key-to-change-595ca4db3a2948c6b44569b58d530c8c)
- [Enablement Deck](https://docs.google.com/presentation/d/1J9WJEcijyz0I1VbU22v28ZQBAWpZnWNQqaGKwoMY5CI/edit?usp=sharing)

## Contributing: 
- Make sure you've read through the accessibility and Amplify Background above
-  Pick a screen thread - each screen below has context of the User Story
- Add suggestions on copy in a comment
- Upvote comments that have helpful suggestions

# Goal: 
The user journey assumes that most of the education on the campaign is happening from the advocacy groups and constituents are just following up on the call to action on our platform 
- [Browse cause](https://github.com/ProgramEquity/amplify-front-end/discussions/79#discussioncomment-2150048): home page, select campaign 
- [Browse Reps](https://github.com/ProgramEquity/amplify-front-end/discussions/79#discussioncomment-2150070): select one in your zipcode
- [Review Letter](https://github.com/ProgramEquity/amplify-front-end/discussions/79#discussioncomment-2150109): customize your letter
- [Validate address](https://github.com/ProgramEquity/amplify-back-end/discussions/137#discussioncomment-2146539): sign name and address/get updates 
- [Select donation](https://github.com/ProgramEquity/amplify-front-end/discussions/79#discussioncomment-2149143): add the amount to donate
- [Action complete](https://github.com/ProgramEquity/amplify-front-end/discussions/79#discussioncomment-2150154): celebrate the change you made and invite others

**Comment by manishapriya94** [comment]
## Donations and Payment
<img width="544" alt="Screen Shot 2022-02-10 at 4 23 55 AM" src="https://user-images.githubusercontent.com/9143339/153408092-cf4f22f7-ec63-490e-874d-6175f858a8d5.png">

**Paying to send letter offsets open source dev** 
The base cost of the letter helps us pay for all operations (1.00 per letter pays for our use of lob, stripe and averages out the other tools we use to keep our development environments going). Overall, we should see these costs go down as we continue to reach out with our sponsors team about our nonprofit status. 

**Donation amounts**
In our initial survey, we saw that a form of civic engagement that was prioritized was mutual aid. Pricepoints listed are to decrease decision paralysis that ‘enter an amount’ donation has. The prices range from 50 (based on an average seen from our larger national orgs) all the way to 2$ as Movement based advocacies recognize all contributions. 

- copy that reflects the value of sending a letter (many advocacy groups cite this is the first time they can attempt this after many years of strapped resources, we could talk about how we want to keep user data private, could also do AB tests?)
- tag that this is powered by a volunteer initiative and link to programequity?

**Reply by manishapriya94** [reply]
Sunrise Focus Group: 
- Blurb or dropdown as to why we need the address for letter as return address and for data?
- Create a different drop down if people want to donate to the cause 
- Reminder that the dollar is for a PHYSICAL letter
- Create a slider and see where 
- Promo code shout out after

**Comment by manishapriya94** [comment]
## Browse Causes
<img width="386" alt="Screen Shot 2022-02-10 at 6 27 56 AM" src="https://user-images.githubusercontent.com/9143339/153427929-f8ecbac2-82c0-4ecc-8a96-0df64361caf0.png">

How can we communicate this concept clearer and why it matters? 
**Functions of Amplify: we are reducing the time it takes to take action..by 3 hours**

🔍 Find a cause you connect with 

🔍 Find your civic engagement network 

📝 Send off your story 

📪 We handle post (no one carries printers, envelopes, or stickers, but you carry your phone)

**Reply by manishapriya94** [reply]
Should there be an outlining of expectations similar to this?
<img width="567" alt="Screen Shot 2022-02-10 at 6 31 10 AM" src="https://user-images.githubusercontent.com/9143339/153428638-7ec7f805-9f15-4e30-ab87-2bee9a3f01e3.png">

**Comment by manishapriya94** [comment]
## Browse Representatives 
This screen helps the user understand how to use the action and a preview of the advocacy. Would a chatbot be more powerful here? Or to hear from one of the advocacy groups in a pop up? 

<img width="388" alt="Screen Shot 2022-02-10 at 6 29 41 AM" src="https://user-images.githubusercontent.com/9143339/153428343-63eb6e93-f479-4802-958a-5597c249fb1c.png">

**Reply by manishapriya94** [reply]
**Sunrise focus feedback:** 
- move search to the right 
- Add a gif to summarize the bill
- letter goal should be here in header
- Auth0 should have disclaimer: "we need to make sure you're a human - let's confirm your phone number"

**Comment by manishapriya94** [comment]
## Review Letter 
<img width="385" alt="Screen Shot 2022-02-10 at 6 31 54 AM" src="https://user-images.githubusercontent.com/9143339/153428772-8c992397-25ae-4ee2-8418-c608b2a93edc.png">
We might want to add copy that makes it more compelling to add their story, for example it gives insights to specifc contexts that a public official might not experience ever 
"At the end of the day, we use data to open the door but its stories that change our leaders hearts" - Nikki Dominguez, AAAJ

**Reply by manishapriya94** [reply]
Sunrise Focus group feedback: 
- Add website of representative 
- Add a select representative instruction 
  - make button less bold until selected
- Change button before authentication to say "log into edit"

**Comment by manishapriya94** [comment]
## Action complete? 
Which screen is more powerful to you? 

<img width="218" alt="Screen Shot 2022-02-10 at 6 37 53 AM" src="https://user-images.githubusercontent.com/9143339/153430218-cd70a71e-ef2e-4dcb-a3e1-91a2f664373e.png">


- What would you want to know once you sent your letter?
- 1.5% letters do have a fail rate, how should we communicate this and that they can get a refund by emailing transaction id to support 
- How do we create incentives towards keeping up momentum?

**Reply by omaciel** [reply]
I like the first example best--the one with the map--because it visually shows how I am part of a collaborative effort and the impact that I'm helping with.
